@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/hellfire/fn_setLasercode.sqf

```diff
@@ -0,0 +1,75 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_setLaserCode
+
+Description:
+	Setting of Ace laser code
+
+Parameters:
+    _heli - the helicopter
+    _channel - the channel Letter for the desired channel
+    _code the code you wish to set it to
+
+Returns:
+	Returns - "invalid" if it is considered an invalid ace code
+
+Examples:
+    Success = [_heli, "A", "1112"] call fza_fnc_setLaserCode;
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli", "_Channel","_newCode"];
```

Comment:
With the new variable layout, this function may not be needed. We will however need to do some validation, but that will likely be done when the KU is entering it.